### PR TITLE
Static extensions

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: public/extensions/statics.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+injections:
```
Comment (mrCherry97) [inline public/extensions/statics.yaml:1]: Move this to examples

File: resources/web/configmap.yaml [hunk 2]
```diff
@@ -2,68 +2,14 @@ kind: ConfigMap
...
-            "config": {
-              "kubeconfigUrl": "/kubeconfig"
-            }
-          },
-          "GZIP": {
-            "isEnabled": true
-          },
-          "VISUAL_RESOURCES": {
+        "features": {
+          "EXTENSIBILITY_INJECTIONS": {
```
Comment (mrCherry97) [inline resources/web/configmap.yaml:12]: Revert this removed code

**Review by mrCherry97 — CHANGES_REQUESTED** [review]

File: docs/extensibility/statics.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- [Overview](#overview)
+- [Static extension label](#static-extension-label)
+- [Extension version](#extension-version)
+- [_general_ section](#general-section)
+- [_injections_ section](#injections-section)
+
+## Overview
+
+This document describes the required ConfigMap setup that you need to configure in order to handle a static extension.
```
Comment (mmitoraj) [inline docs/extensibility/statics.md:13]:
```suggestion
This document describes the required ConfigMap setup that you need to configure to handle a static extension.
```

File: docs/extensibility/statics.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+You can provide all the ConfigMap data sections as either JSON or YAML.
+
+## Static extension label
+
+To define static extension add label `busola.io/extension:statics` to ConfigMap.
```
Comment (mmitoraj) [inline docs/extensibility/statics.md:18]:
```suggestion
To define a static extension, add the `busola.io/extension:statics` label to the ConfigMap.
```

File: docs/extensibility/statics.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+## Extension version
+
+The version is a string value that defines in which version the extension is configured. It is stored as a value of the `busola.io/extension-version` label. When created manually, use the latest version number, for example, `'0.5'`.
+
+> **NOTE:**: Busola supports only the two latest versions of the configuration. Whenever a new version of the configuration is proposed, go to your Extension and migrate your configuration to the latest version.
```
Comment (mmitoraj) [inline docs/extensibility/statics.md:24]:
```suggestion
> **NOTE:** Busola supports only the two latest versions of the configuration. Whenever a new version of the configuration is proposed, go to your Extension and migrate your configuration to the latest version.
```

File: docs/extensibility/statics.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+## _general_ section
+
+The **general** section is not required as static extensions present data that are not conected to any resource. Instead they may use information from page they are displayed on via variable `$embedResource`.
```
Comment (mmitoraj) [inline docs/extensibility/statics.md:28]:
```suggestion
The **general** section is not required as static extensions present data that are not connected to any resource. Instead, they may use information from the page they are displayed on via variable `$embedResource`.
```

File: docs/extensibility/statics.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+## _injections_ section
+
+For more information read [widget _injections_ overview](display-section.md#widget-injections-overview).
```
Comment (mmitoraj) [inline docs/extensibility/statics.md:32]:
```suggestion
For more information read the [widget injections overview](display-section.md#widget-injections-overview).
```

File: examples/statics/kibana-links.yaml [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+metadata:
+  name: kibana-links
+  namespace: kube-public
+  labels:
+    app.kubernetes.io/name: kibana-links
+    busola.io/extension: statics
+    busola.io/extension-version: '0.5'
+data:
+  injections: |-
+    - name: "Logs for namespace"
```
Comment (mmitoraj) [inline examples/statics/kibana-links.yaml:12]:
```suggestion
    - name: "Logs for Namespace"
```
should we capitalize it here as in our docs?

File: examples/statics/kibana-links.yaml [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+        {'namespace_name': $embedResource.metadata.name}
+      )"
+      widget: ExternalLink
+      type: button
+      copyable: false
+      targets:
+        - slot: details-header
+          location: Namespaces
+      order: 1 
+    - name: "Logs for pod"
```
Comment (mmitoraj) [inline examples/statics/kibana-links.yaml:25]:
```suggestion
    - name: "Logs for Pod"
```
should we capitalize it here as in our docs?

**Review by mmitoraj — COMMENTED** [review]
Please see my comments below :)

**Review by mrCherry97 — APPROVED** [review]

**Review by mmitoraj — DISMISSED** [review]

**Pull request closed without merging**